yt_dlp/extractor/playerfm.py

```diff
@@ -0,0 +1,60 @@
+
+from .common import InfoExtractor
+from ..utils import clean_html, int_or_none, url_or_none
+from ..utils.traversal import traverse_obj
+
+
+class PlayerFmIE(InfoExtractor):
+    _VALID_URL = r'(?P<url>https?://(?:www\.)?player\.fm/(?:series/)?[\w-]+/(?P<id>[\w-]+))'
+    _TESTS = [{
+        'url': 'https://player.fm/series/chapo-trap-house/movie-mindset-33-casino-feat-felix',
+        'info_dict': {
+            'id': 'movie-mindset-33-casino-feat-felix',
+            'thumbnail': r're:^https://.*\.(jpg|png)',
+            'title': 'Movie Mindset 33 - Casino feat. Felix',
+            'creators': ['Chapo Trap House'],
+            'description': r're:The first episode of this season of Movie Mindset is free .+ we feel about it\.',
+            'duration': 6830,
+            'ext': 'mp3',
+        },
+    }, {
+        'url': 'https://player.fm/series/nbc-nightly-news-with-lester-holt/thursday-april-24-2025',
+        'info_dict': {
+            'id': 'thursday-april-24-2025',
+            'thumbnail': r're:^https://.*\.(jpg|png)',
+            'title': 'Thursday, April 24, 2025',
+            'creators': ['Lester Holt, NBC News'],
+            'description': '<p>Trump urges Putin to \'STOP\' after deadly Russian strike on Ukraine; More than 90,000 people have paid their respects to Pope Francis; Miami shooting leaves three injured; and more on tonight’s broadcast.</p>',
+            'duration': 1250,
+            'ext': 'mp3',
+        },
+    }, {
+        'url': 'https://player.fm/series/soccer-101/ep-109-its-kicking-off-how-have-the-rules-for-kickoff-changed-what-are-the-best-approaches-to-getting-the-game-underway-and-how-could-we-improve-on-the-present-system',
+        'info_dict': {
+            'id': 'ep-109-its-kicking-off-how-have-the-rules-for-kickoff-changed-what-are-the-best-approaches-to-getting-the-game-underway-and-how-could-we-improve-on-the-present-system',
```

Comment:
This URL is dead? And can we use anything else for the `id`? This id is way too long